### PR TITLE
tox + pass flake8

### PR DESCRIPTION
**Pull request by hashar**
Add a tox entry point to easily run flake8
Make all files pass flake8 with python3.4
List dependencies in requirements.txt for easily installation
Add a tox 'venv' that let us easily run arbitrary commands in the env

### CODE REVIEW

**Comment by hashar** [comment]
I have added https://github.com/hashar/tools-ci/commit/8515c4fabc72b8a83954e941b7984a83b179612c  to be able to:  `tox -e venv -- python build_table.py`

**Comment by legoktm** [comment]
Er, I got the number wrong >.<

**Comment by legoktm** [comment]
I added a LICENSE file to the repo. Please confirm you're ok with the license I picked (or let's figure out a different one) and then I'll merge your PRs \o/

**Comment by hashar** [comment]
I have signed the LICENSE at https://github.com/legoktm/tools-ci/commit/36fad28ee2a15a24669780231cb466af4eb49f4f#commitcomment-13106575  :D

File: build_table.py [hunk 1]
```diff
@@ -57,15 +58,15 @@ def read_npm(self, path, github_name):
                     self.data[github_name][job]['version'] = version
 
     def read_zuul(self, info, github_name):
-#        if github_name != 'mediawiki-extensions-Echo':
-#            return
+        #if github_name != 'mediawiki-extensions-Echo':
```
Comment (legoktm) [inline build_table.py:61]: We should kill this block entirely, I don't remember what it's for, probably some debugging thing :/

File: build_table.py [hunk 2]
```diff
@@ -165,8 +180,8 @@ def read_zuul(self, info, github_name):
 paths.remove('mediawiki')
 paths = ['mediawiki'] + paths
 for repo_path in paths:
-#    if repo_path != 'mediawiki-extensions-Echo':
-#        continue
+    #if repo_path != 'mediawiki-extensions-Echo':
```
Comment (legoktm) [inline build_table.py:183]: This too :P

**Pull request merged**